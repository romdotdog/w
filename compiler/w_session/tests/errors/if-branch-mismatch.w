fn main(a: u32, b: f32, c: f64): u32 {
    if a {
        b
    } else {
        c
    }
}