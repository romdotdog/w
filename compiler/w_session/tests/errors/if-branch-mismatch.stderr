tests/errors/if-branch-mismatch.w:7:1: error: if statement cannot have a result
}
^
